__tests__/actions/major.test.ts

```diff
@@ -280,4 +282,72 @@ describe("Major actions", () => {
       majorResult,
     );
   });
+
+  it("should return the available catalog years", async () => {
+    // Create 4 majors
+    await prisma.major.create({
+      data: {
+        catalogYear: "2020-2021",
+        name: "Test Major 1",
+        programType: ProgramType.Major,
+      },
+    });
+
+    await prisma.major.create({
+      data: {
+        catalogYear: "2021-2022",
+        name: "Test Major 1",
+        programType: ProgramType.Major,
+      },
+    });
+
+    await prisma.major.create({
+      data: {
+        catalogYear: "2022-2023",
+        name: "Test Major 1",
+        programType: ProgramType.Major,
+      },
+    });
+
+    await prisma.major.create({
+      data: {
+        catalogYear: "2023-2024",
+        name: "Test Major 1",
+        programType: ProgramType.Major,
+      },
+    });
+
+    const catalogYearsObjs = await getCatalogYears();
+    const catalogYears = catalogYearsObjs.map((year) => year.catalogYear);
+
+    expect(catalogYears).toContain("2020-2021");
+    expect(catalogYears).toContain("2021-2022");
+    expect(catalogYears).toContain("2022-2023");
+    expect(catalogYears).toContain("2023-2024");
+  });
+
+  it("should return the available catalog years for a specific major", async () => {
```

Comment:
> I believe the `afterAll` function calls `prisma.user.deleteMany()` which deletes all the entries in the users table.

In a comment below I address this. I used the same conventions as already existing in the file where all database data is deleted after the test suite. I think there is a need for a test refactor that creates and deletes specific ids then clearing the whole database. Also may be nice to standardized the CRUD operations by creating uniform server actions.

Reply:
Actually this test suite already has an `afterEach` lifecycle hook that clears majors between tests so my mistake. Having standardized CRUD operations is a good idea, but I fear that it might lead to a lot of boilerplate-y code when we only need a subset of operations. Moreover, we don't have an API layer (yet) for which we need to standardize CRUD ops and can leverage the ad-hoc and customizable nature of server actions for now. We might need to do this for the `Permissions` table though because we were planning to create an API for that but that's kind of unrelated right now.

Reply:
sounds good